tests/Fixtures/Integration/misc/class_attributes_separation,braces.test

```diff
@@ -0,0 +1,25 @@
+--TEST--
```

Comment:
now I got lost, this one is in `misc`, and on the other side it's listed as missing `prio` test

Reply:
My bad, if removed it from the list of to check priority checkers, and removed it as missing a test